src/components/MovieList/MovieCard.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from "react";
+import Cards from "./Cards";
+import PNF from "./PNF";
+import { CardDivStyle } from "../Styles/Card.style";
+import { apiUrl, searchUrl } from "../ constants/global";
+
+const MovieCard = ({ input }) => {
+  const [data, setData] = useState([]);
```

Comment:
State names must be meaningful. Keep the habit of giving meaningful names for each and every variable/state.